go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-state-metrics v1.7.2
 	k8s.io/kubectl v0.18.2
+	rsc.io/letsencrypt v0.0.3 // indirect
```

Comment:
I think we do not need the indirect one. could you remove it before merge? 
PS.: Unfortunately, go mod tidy has not removed all indirect additions. To check we can run ` go mod why` and then if in the msg we see `main module does not need package` we can remove it.